index.html

```diff
@@ -0,0 +1,147 @@
+                    </li>
+                    <li>
+                        <a href="">Logout</a>
+                    </li>
+                </ul>
+            </div>
+            <nav class="bottom-navbar">
+                <div class="logo">
+                    <a href="index.html">
+                        <span>Zi</span>
+                        RO
+                        <br>
+                        <SPan>Music</SPan>
+                    </a>
+                </div>
+                <ul>
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#join">Join</a>
+                    </li>
+                    <li>
+                        <a href="#partners">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+    <!-- MAIN SECTION STARTING -->
+    <main>
+        <section class="main">
+            <nav class="Mobile-navbar">
+                <img id="cancelBtn" src="images/Icon - Cancel.svg" alt="">
+                <i id="hamburger" class="fas fa-bars hamburger" aria-hidden="true"></i>
+                <ul id="hamburger-list">
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="">Join</a>
+                    </li>
+                    <li>
+                        <a href="">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="container">
+                <div class="content">
+                    <p class="content-hellow">"Hellow! Music World"</p>
+                    <h1 class="main-content-heading">ZIRO MUSIC FESTIVAL NIGHT 2023</h1>
+                    <p class="content-detail">A Ziro Music Festival is a live music performance in front of an audience. 
+                        The performance may be by a single musician, sometimes then called a recital, 
+                        or by a musical ensemble, such as an orchestra, choir, or band. 
+                    </p>
+                    <h2 class="content-secondHeading">2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p class="content-lastline">@ National Music of universe.</p>
+                </div>
+        </div>
+        </section>
```

Comment:
Kindly use the same color, font-size, and font-family as in the provided design.  Same goes for the mobile version

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/9c846ad0-1bd1-452d-814f-e76622bf2a8d)| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/1ba1a62d-b5c8-42cf-bc2f-4174e15734b0) |

Reply:
Fixed.